Formula/supermodel.rb

```diff
@@ -17,7 +17,8 @@ class Supermodel < Formula
   depends_on "sdl"
 
   def install
-    inreplace "Makefiles/Makefile.SDL.OSX.GCC" do |s|
+    makefile_dir = build.head? ? Makefiles/Makefile.OSX : Makefiles/Makefile.SDL.OSX.GCC
```

Comment:
```suggestion
    makefile_dir = build.head? ? "Makefiles/Makefile.OSX" : "Makefiles/Makefile.SDL.OSX.GCC"
```

Reply:
Oops, I forgot the quotes in my example. :flushed: Please accept those changes, or tick the "allow maintainers to push" box in the corner of this PR, then this should pass CI.